package.json

```diff
@@ -52,7 +52,7 @@
   "homepage": "https://github.com/redstone-finance/redstone-smartweave#readme",
   "dependencies": {
     "@weavery/clarity": "^0.1.5",
-    "arweave": "^1.10.16",
+    "arweave": "1.10.18",
```

Comment:
@asiaziola , the arweave 1.10.23 (or basically anything above 1.10.18) still doesn't seem to be compatible with ArLocal (tested with latest ArLocal 1.1.20) - it is till throwing this "offset" error while loading transaction data.
@cedriking

Reply:
https://github.com/textury/arlocal/issues/63